kustomize/overlays/stable/manager/patches/deploy-config.yaml

```diff
@@ -11,4 +11,6 @@ data:
   EVENT_BRIDGE_SSO_URL: "https://keycloak-event-bridge-stable.apps.ob-stable.0b9l.p1.openshiftapps.com/auth/realms/event-bridge-fm"
   EVENT_BRIDGE_SSO_CLIENT_ID: "event-bridge"
   EVENT_BRIDGE_KAFKA_SECURITY_PROTOCOL: SASL_SSL
-  EVENT_BRIDGE_KAFKA_TOPIC_PREFIX: "ob-stable-"
\ No newline at end of file
+  # TODO: remove the following property after the system is updated to use "event-bridge.resource-prefix" property
```

Comment:
do we have a JIRA opened for this?

Reply:
I created it: https://issues.redhat.com/browse/MGDOBR-475